apps/recnet-api/src/database/repository/rec.repository.ts

```diff
@@ -12,6 +12,15 @@ import { getNextCutOff } from "@recnet/recnet-date-fns";
 export default class RecRepository {
   constructor(private readonly prisma: PrismaConnectionProvider) {}
 
+  public async findRecById(recId: string): Promise<Rec | null> {
+    return this.prisma.recommendation.findUnique({
```

Comment:
[nit] Maybe you can use `findUniqueOrThrow` if this is not a normal use case. But the error handling in `recnet.exception.filter.ts` should be modified a bit.

Reply:
I see. I originally use `findUniqueOrThrow` but when testing, the error message I saw was "User not found". I forget to modify `recnet.exception.filter.ts`. No wonder.